qemu/qemu.go

```diff
@@ -875,7 +875,9 @@ func (vhostuserDev VhostUserDevice) QemuParams(config *Config) []string {
 		devParams = append(devParams, string(driver))
 		devParams = append(devParams, fmt.Sprintf("chardev=%s", vhostuserDev.CharDevID))
 		devParams = append(devParams, fmt.Sprintf("tag=%s", vhostuserDev.Tag))
-		devParams = append(devParams, fmt.Sprintf("cache-size=%dM", vhostuserDev.CacheSize))
+		if vhostuserDev.CacheSize != 0 {
+			devParams = devParams = append(devParams, fmt.Sprintf("cache-size=%dM", vhostuserDev.CacheSize))
```

Comment:
So this commit isn't building.  I guess we need to remove the second 'devParams = '

Reply:
Wow, sorry for the typo. Should be updated already.